### ISSUE TITLE
Cluster without adding cluster properties

### ISSUE THREAD

**Issue by pathmapper** [mapbox/tippecanoe#815]
When clustering, the properties `clustered`, `point_count` and  `sqrt_point_count` are automatically added to the remaining features.

It would be good to have an option to prevent this per property to keep the tilesizes small if one or more of these are not needed. `-x` (and also `-X)` are not working for this purpose.

**Comment by ericfischer** [comment]
Good point. I should make `-x` apply to these synthetic attributes too.
